src/main/java/eu/phiwa/dragontravel/core/listeners/PlayerListener.java

```diff
@@ -252,6 +252,11 @@ public void onPlayerDamage(EntityDamageEvent event) {
         if (player.hasPermission("dt.ignoredamagerestriction"))
             return;
 
+        if (DragonManager.getDragonManager().isInDismountedList(player)) {
```

Comment:
I guess, this should be moved before the previous check, otherwise players with the permission "dt.ignoredamagerestriction" will not be prevented from taking damage.